assets/css/style.css

```diff
@@ -0,0 +1,288 @@
+/******************************************************************************
+FSWD:  Christopher B. Zenner
+Date:  02/15/2020
+File:  style.css
+Ver.:  0.1.0 20200214
+
+This CSS specifies the layout and style for the basic portfolio site of the
+FSWB.
+*******************************************************************************/
+
+/* Debug */
+/* { border: 1px solid red !important; }
+   { border: 1px solid orange !important; }
+   { border: 1px solid yellow !important; }
+   { border: 1px solid green !important; }
+   { border: 1px solid blue !important; }
+   { border: 1px solid purple !important; }
+*/
+
+/******************************************************************************
+ Global Elements
+*******************************************************************************/
+* {
+  /* make widths static */
+  box-sizing: border-box;
+}
+
+body {
+  background: #eee;
+  /* Image options for backgrounds:
+  background-image: url("../img/dot-grid.png");
+  background-image: url("../img/grey.png"); */
+  background-image: url("../img/grey_@2X.png");
+  color: #777;
+  font: 16px/1.5em 'Open Sans', sans-serif;
+}
+
+/******************************************************************************
+ Header Elements
+*******************************************************************************/
+header {
+  letter-spacing: 3px;
+}
+
+header, 
+.main-container {
+  /* DEBUG:
+     border: 1px red solid; */ 
+  margin: 0 auto;
+  width: 960px;
+}
+
+header,
+h1 {
+  font-family: 'Raleway', sans-serif;
+}
+
+/******************************************************************************
+ Navigation Elements
+*******************************************************************************/
+nav {
+  float: right;
+  width: 30%;
+}
+
+nav ul {
+  float: right;
+  list-style-type: none;
+  margin: 2em 0 0 0;
+  overflow: hidden;
+  padding: 0;
+}
+
+nav ul li {
+  border-right: 1px solid #ccc;
+  float: left;
+}
+
+nav ul li:last-child {
+  border-right: none;
+}
+
+nav ul li a {
+  color: #777;
+  display: block;
+  font-size: 0.8em;
+  line-height: 1em;
+  padding: 3px 10px;
+  text-align: center;
+}
+
+nav ul li a:active,
+nav ul li a:hover {
+  /* Underline is more aestethically pleasing here. */
+  /* background: #ddd; */
+}
+
+nav ul li a:visited {
+  color: #aaa;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:active,
+a:hover {
+  text-decoration: underline;
+}
+
+/******************************************************************************
+ Main Content Elements
+*******************************************************************************/
+main {
+  background: #fff;
+  border: 1px solid #ddd;
+  padding: 0 20px;
+}
+
+h1,
+main a {
+  color: #4aaaa5;
+}
+
+h1 {
+  border-bottom: 3px #ddd solid;
+  padding: 10px 0;
+}
+
+main img {
+  /* ISSUE: only rounding top, left corner (20200215). */
```

Comment:
FYI this is because of the padding. With the padding turned off and the border-radius:

<img width="363" alt="image" src="https://user-images.githubusercontent.com/661038/74705492-ee354900-51c8-11ea-846d-d90f9388de4a.png">

You could fix it by using margin instead.